src/Server.js

```diff
@@ -23,6 +23,7 @@ class Server extends EventEmitter {
    * @typedef {Object} ServerArgs
    * @property {string} ServerArgs.address - The server address/name.
    * @property {string} [ServerArgs.port=null] - The server port.
+   * @property {Object} ServerArgs.headers - The headers to be used in requests.
```

Comment:
One last thing, can you make this 
* @property {Object} [ServerArgs.headers={}] - The headers to be used in requests.

This just documents the property as optional with default.